gammapy/cube/models.py

```diff
@@ -340,3 +341,48 @@ def evaluate(self, lon, lat, energy):
     def copy(self):
         """A shallow copy"""
         return copy.copy(self)
+
+
+class BackgroundModel(Model):
+    """Background model
+
+    Parameters
+    ----------
+    background : `~gammapy.maps.Map`
+        Background model map
+    norm : float
```

Comment:
Can you change `float` to `~astropy.units.Quantity` here? It does not really matter, but for consistency with the other model classes we have...

Reply:
Well the `SkyDiffuseCube` takes a `float` instead of `Quantity`. And this is just a number, so...?

Reply:
I guess it's a matter of taste. For dimensionless quantities it does not really matter. I'm also fine with keeping `float`. At some point we should just make it uniform. I think for our spectral models we still use `Quantity` for the index...do as you like.

Reply:
I think I had a discussion with @registerrier some time back when he said that calculations were much slower when the index was a `Quantity` as compared to when it was a `float`.
Well, since this code is a temporary solution, maybe we let this be as is for now...